ios/CodePush/CodePush.m

```diff
@@ -247,6 +247,20 @@ - (instancetype)init
  */
 - (void)initializeUpdateAfterRestart
 {
+    dispatch_async(dispatch_get_main_queue(), ^{
+        if ([_bridge.bundleURL.scheme hasPrefix:@"http"]) {
```

Comment:
does `hasPrefix` do ignoreCase comparison? Do you think the casing could be an issue?